cloudberry/zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/TestQuery.scala

```diff
@@ -4,20 +4,25 @@ import edu.uci.ics.cloudberry.zion.model.schema.{GeoCellThousandth, _}
 import org.joda.time.{DateTime, DateTimeZone}
 import play.api.libs.json.Json
 
+
 object TestQuery {
 
   DateTimeZone.setDefault(DateTimeZone.UTC)
   val TwitterDataSet = TwitterDataStore.DatasetName
+  val TwitterDataSetForSQL = TwitterDataStoreForSQL.DatasetName
```

Comment:
I've tried to write a trait/class and two extends, but the code seems ugly(a lot of "lazy"s), so added these val for SQLGeneratorTest instead.